bot/core/bot.py

```diff
@@ -44,7 +51,13 @@ async def on_ready(self) -> None:
             )
             await self.log_channel.send(embed=embed)
 
-        print("Bot is ready")
+        print(colorama.Fore.GREEN + "Bot is ready")
+
+    async def shutoff(self) -> None:
```

Comment:
We should probably stick to `close` since that's the name in `Base_Bot` class and it is what things will call if they need to close the bot, that way if they call `close` it would fall back to the original `Base_Bot` instead of going here, since this doesn't override that function anymore, there is no real reason to rename this.

Note that by reverting this, you'll also need to revert renamed references to this function